src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.exceptions.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("loogiin");
+        user.setPassword("password");
+        user.setAge(20);
+    }
+
+    @Test
+    void register_Ok() {
+        assertTrue(registrationService.register(user) == user);

```

Comment:
we have assertEquals for this

Reply:
you also need to make sure not only that return value is correct, but that is was added to the storage

Reply:
How can I do this?
Write a method of getting the length of the db in the registration service - it is not logical.
It is also impossible to access the db field from the outside